tensor_api.ipynb

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
<p>What are the availbe types? List them and explain them. Similar to <a href="https://pytorch.org/docs/stable/tensors.html" target="_blank">https://pytorch.org/docs/stable/tensors.html</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/intel-isl/Open3D/pull/2067/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>